qiskit/providers/aer/noise/utils/noise_transformation.py

```diff
@@ -212,24 +216,65 @@ def approximate_noise_model(model, *,
     approx_noise_model._basis_gates = model._basis_gates
     return approx_noise_model
 
+def pauli_operators():
+    pauli_1_qubit = {
+                    'X': [{'name': 'x', 'qubits': [0]}],
+                    'Y': [{'name': 'y', 'qubits': [0]}],
+                    'Z': [{'name': 'z', 'qubits': [0]}]
+                    }
+    pauli_2_qubit = {
+        'XI': [{'name': 'x', 'qubits': [0]}, {'name': 'id', 'qubits': [1]}],
```

Comment:
Qubit 0 and 1 need to be swapped here. The convention used in other functions is eg:

```
pauli_error([('XY', 1.0)]) --> QuantumError([([{'name': 'y', 'qubits': [0]}, {'name': 'x', 'qubits': [1]}], 1.0)])
```